[BUG] OpenSearch v1.2.4 Docker image is broken while v1.2.3 just works

This command works
![Captura de 2022-01-21 17-47-07](https://user-images.githubusercontent.com/7274874/150566732-c068a98d-56b1-4139-841d-30e08aec4534.png)

But this one doesn't

![Captura de 2022-01-21 17-48-23](https://user-images.githubusercontent.com/7274874/150567159-5421a89b-e3cc-4b7e-9143-310a7e749b25.png)

Errors are shown below

![Captura de 2022-01-21 17-49-08](https://user-images.githubusercontent.com/7274874/150567163-4f478217-a024-405f-8cfc-1d50c68d5934.png)
![Captura de 2022-01-21 17-49-28](https://user-images.githubusercontent.com/7274874/150567166-8a00d63c-3a45-496b-bc8a-8888923113ad.png)
![Captura de 2022-01-21 17-49-47](https://user-images.githubusercontent.com/7274874/150567167-e5ccff6e-2d08-4550-b791-849f44880e1a.png)
![Captura de 2022-01-21 17-50-06](https://user-images.githubusercontent.com/7274874/150567171-d19d693e-4734-47ee-bd23-33e020cfb385.png)

Thanks a lot for you patience.

I'm running a last-minute updated Fedora 35 system (kernel v5.15.15, podman v3.4.4)

If I run _podman container run --rm --name=puto -e "discovery.type=single-node" -p 9200:9200 opensearchproject/opensearch:latest_ I get:

```
Resolved "opensearchproject/opensearch" as an alias (/home/usuari/.cache/containers/short-name-aliases.conf)
Trying to pull docker.io/opensearchproject/opensearch:latest...
Getting image source signatures
Copying blob 906ed3bd695c done  
Copying blob 1682fbe93720 done  
Copying blob 4f11d5532c74 done  
Copying blob 3a461b3ae562 done  
Copying blob 281f706dd97a done  
Copying blob 2f071fa29f89 done  
Copying config 9ae13ad440 done  
Writing manifest to image destination
Storing signatures
Enabling execution of install_demo_configuration.sh for OpenSearch Security Plugin
OpenSearch Security Demo Installer
 ** Warning: Do not use on production or public reachable systems **
Basedir: /usr/share/opensearch
OpenSearch install type: rpm/deb on NAME="Amazon Linux"
OpenSearch config dir: /usr/share/opensearch/config
OpenSearch config file: /usr/share/opensearch/config/opensearch.yml
OpenSearch bin dir: /usr/share/opensearch/bin
OpenSearch plugins dir: /usr/share/opensearch/plugins
OpenSearch lib dir: /usr/share/opensearch/lib
Detected OpenSearch Version: x-content-1.2.4
Detected OpenSearch Security Version: 1.2.4.0

### Success
### Execute this script now on all your nodes and then start all nodes
### OpenSearch Security will be automatically initialized.
### If you like to change the runtime configuration 
### change the files in ../securityconfig and execute: 
"/usr/share/opensearch/plugins/opensearch-security/tools/securityadmin.sh" -cd "/usr/share/opensearch/plugins/opensearch-security/securityconfig" -icl -key "/usr/share/opensearch/config/kirk-key.pem" -cert "/usr/share/opensearch/config/kirk.pem" -cacert "/usr/share/opensearch/config/root-ca.pem" -nhnv
### or run ./securityadmin_demo.sh
### To use the Security Plugin ConfigurationGUI
### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/admin.
### (Ignore the SSL certificate warning because we installed self-signed demo certificates)
Enabling OpenSearch Security Plugin
[2022-01-21T20:33:39,054][INFO ][o.o.n.Node               ] [f69642b92937] version[1.2.4], pid[100], build[tar/e505b10357c03ae8d26d675172402f2f2144ef0f/2022-01-14T03:38:06.881862Z], OS[Linux/5.15.15-200.fc35.x86_64/amd64], JVM[AdoptOpenJDK/OpenJDK 64-Bit Server VM/15.0.1/15.0.1+9]
[2022-01-21T20:33:39,059][INFO ][o.o.n.Node               ] [f69642b92937] JVM home [/usr/share/opensearch/jdk], using bundled JDK [true]
[2022-01-21T20:33:39,073][INFO ][o.o.n.Node               ] [f69642b92937] JVM arguments [-Xshare:auto, -Dopensearch.networkaddress.cache.ttl=60, -Dopensearch.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -XX:+ShowCodeDetailsInExceptionMessages, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dio.netty.allocator.numDirectArenas=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Djava.locale.providers=SPI,COMPAT, -Xms1g, -Xmx1g, -XX:+UseG1GC, -XX:G1ReservePercent=25, -XX:InitiatingHeapOccupancyPercent=30, -Djava.io.tmpdir=/tmp/opensearch-16540438191390975168, -XX:+HeapDumpOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Dclk.tck=100, -Djdk.attach.allowAttachSelf=true, -Djava.security.policy=/usr/share/opensearch/plugins/opensearch-performance-analyzer/pa_config/opensearch_security.policy, -Dopensearch.cgroups.hierarchy.override=/, -XX:MaxDirectMemorySize=536870912, -Dopensearch.path.home=/usr/share/opensearch, -Dopensearch.path.conf=/usr/share/opensearch/config, -Dopensearch.distribution.type=tar, -Dopensearch.bundled_jdk=true]
[2022-01-21T20:33:40,971][INFO ][o.o.s.s.t.SSLConfig      ] [f69642b92937] SSL dual mode is disabled
[2022-01-21T20:33:40,972][INFO ][o.o.s.OpenSearchSecurityPlugin] [f69642b92937] OpenSearch Config path is /usr/share/opensearch/config
[2022-01-21T20:33:41,468][INFO ][o.o.s.s.DefaultSecurityKeyStore] [f69642b92937] JVM supports TLSv1.3
[2022-01-21T20:33:41,470][INFO ][o.o.s.s.DefaultSecurityKeyStore] [f69642b92937] Config directory is /usr/share/opensearch/config/, from there the key- and truststore files are resolved relatively
[2022-01-21T20:33:41,571][ERROR][o.o.b.OpenSearchUncaughtExceptionHandler] [f69642b92937] uncaught exception in thread [main]
org.opensearch.bootstrap.StartupException: java.lang.IllegalStateException: failed to load plugin class [org.opensearch.security.OpenSearchSecurityPlugin]
	at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:182) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.execute(OpenSearch.java:169) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:100) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138) ~[opensearch-cli-1.2.4.jar:1.2.4]
	at org.opensearch.cli.Command.main(Command.java:101) ~[opensearch-cli-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:135) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:101) ~[opensearch-1.2.4.jar:1.2.4]
Caused by: java.lang.IllegalStateException: failed to load plugin class [org.opensearch.security.OpenSearchSecurityPlugin]
	at org.opensearch.plugins.PluginsService.loadPlugin(PluginsService.java:790) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundle(PluginsService.java:726) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundles(PluginsService.java:528) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.<init>(PluginsService.java:194) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:396) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:319) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.setup(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:412) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178) ~[opensearch-1.2.4.jar:1.2.4]
	... 6 more
Caused by: java.lang.reflect.InvocationTargetException
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:481) ~[?:?]
	at org.opensearch.plugins.PluginsService.loadPlugin(PluginsService.java:781) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundle(PluginsService.java:726) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundles(PluginsService.java:528) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.<init>(PluginsService.java:194) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:396) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:319) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.setup(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:412) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178) ~[opensearch-1.2.4.jar:1.2.4]
	... 6 more
Caused by: org.opensearch.OpenSearchException: plugins.security.ssl.transport.keystore_filepath or plugins.security.ssl.transport.server.pemcert_filepath and plugins.security.ssl.transport.client.pemcert_filepath must be set if transport ssl is requested.
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.initTransportSSLConfig(DefaultSecurityKeyStore.java:422) ~[?:?]
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.initSSLConfig(DefaultSecurityKeyStore.java:258) ~[?:?]
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.<init>(DefaultSecurityKeyStore.java:179) ~[?:?]
	at org.opensearch.security.ssl.OpenSearchSecuritySSLPlugin.<init>(OpenSearchSecuritySSLPlugin.java:218) ~[?:?]
	at org.opensearch.security.OpenSearchSecurityPlugin.<init>(OpenSearchSecurityPlugin.java:252) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:481) ~[?:?]
	at org.opensearch.plugins.PluginsService.loadPlugin(PluginsService.java:781) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundle(PluginsService.java:726) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.loadBundles(PluginsService.java:528) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.plugins.PluginsService.<init>(PluginsService.java:194) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:396) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.node.Node.<init>(Node.java:319) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.setup(Bootstrap.java:242) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:412) ~[opensearch-1.2.4.jar:1.2.4]
	at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178) ~[opensearch-1.2.4.jar:1.2.4]
	... 6 more
uncaught exception in thread [main]
java.lang.IllegalStateException: failed to load plugin class [org.opensearch.security.OpenSearchSecurityPlugin]
Likely root cause: OpenSearchException[plugins.security.ssl.transport.keystore_filepath or plugins.security.ssl.transport.server.pemcert_filepath and plugins.security.ssl.transport.client.pemcert_filepath must be set if transport ssl is requested.]
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.initTransportSSLConfig(DefaultSecurityKeyStore.java:422)
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.initSSLConfig(DefaultSecurityKeyStore.java:258)
	at org.opensearch.security.ssl.DefaultSecurityKeyStore.<init>(DefaultSecurityKeyStore.java:179)
	at org.opensearch.security.ssl.OpenSearchSecuritySSLPlugin.<init>(OpenSearchSecuritySSLPlugin.java:218)
	at org.opensearch.security.OpenSearchSecurityPlugin.<init>(OpenSearchSecurityPlugin.java:252)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
	at org.opensearch.plugins.PluginsService.loadPlugin(PluginsService.java:781)
	at org.opensearch.plugins.PluginsService.loadBundle(PluginsService.java:726)
	at org.opensearch.plugins.PluginsService.loadBundles(PluginsService.java:528)
	at org.opensearch.plugins.PluginsService.<init>(PluginsService.java:194)
	at org.opensearch.node.Node.<init>(Node.java:396)
	at org.opensearch.node.Node.<init>(Node.java:319)
	at org.opensearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:242)
	at org.opensearch.bootstrap.Bootstrap.setup(Bootstrap.java:242)
	at org.opensearch.bootstrap.Bootstrap.init(Bootstrap.java:412)
	at org.opensearch.bootstrap.OpenSearch.init(OpenSearch.java:178)
	at org.opensearch.bootstrap.OpenSearch.execute(OpenSearch.java:169)
	at org.opensearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:100)
	at org.opensearch.cli.Command.mainWithoutErrorHandling(Command.java:138)
	at org.opensearch.cli.Command.main(Command.java:101)
	at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:135)
	at org.opensearch.bootstrap.OpenSearch.main(OpenSearch.java:101)
For complete error details, refer to the log at /usr/share/opensearch/logs/opensearch.log
Killing performance analyzer process 101
OpenSearch exited with code 1
Performance analyzer exited with code 143
```

The output of _podman image ls_ is: 

```
REPOSITORY                              TAG         IMAGE ID      CREATED     SIZE
docker.io/opensearchproject/opensearch  latest      9ae13ad440d3  3 days ago  834 MB
```

**BUT** if I run _podman container run --rm --name=puto -e "discovery.type=single-node" -p 9200:9200 opensearchproject/opensearch:1.2.3_ I get this, instead:

```
Resolved "opensearchproject/opensearch" as an alias (/home/usuari/.cache/containers/short-name-aliases.conf)
Trying to pull docker.io/opensearchproject/opensearch:1.2.3...
Getting image source signatures
Copying blob 8b8a142162d2 done  
Copying blob 5d5ff2b51d5a done  
Copying blob a478f063859a done  
Copying blob f4a5866299c0 done  
Copying blob fa411a7d2505 done  
Copying blob 3099d63a1eeb done  
Copying config e72207c604 done  
Writing manifest to image destination
Storing signatures
Enabling execution of install_demo_configuration.sh for OpenSearch Security Plugin
OpenSearch Security Demo Installer
 ** Warning: Do not use on production or public reachable systems **
Basedir: /usr/share/opensearch
OpenSearch install type: rpm/deb on NAME="Amazon Linux"
OpenSearch config dir: /usr/share/opensearch/config
OpenSearch config file: /usr/share/opensearch/config/opensearch.yml
OpenSearch bin dir: /usr/share/opensearch/bin
OpenSearch plugins dir: /usr/share/opensearch/plugins
OpenSearch lib dir: /usr/share/opensearch/lib
Detected OpenSearch Version: x-content-1.2.3
Detected OpenSearch Security Version: 1.2.3.0

### Success
### Execute this script now on all your nodes and then start all nodes
### OpenSearch Security will be automatically initialized.
### If you like to change the runtime configuration 
### change the files in ../securityconfig and execute: 
"/usr/share/opensearch/plugins/opensearch-security/tools/securityadmin.sh" -cd "/usr/share/opensearch/plugins/opensearch-security/securityconfig" -icl -key "/usr/share/opensearch/config/kirk-key.pem" -cert "/usr/share/opensearch/config/kirk.pem" -cacert "/usr/share/opensearch/config/root-ca.pem" -nhnv
### or run ./securityadmin_demo.sh
### To use the Security Plugin ConfigurationGUI
### To access your secured cluster open https://<hostname>:<HTTP port> and log in with admin/admin.
### (Ignore the SSL certificate warning because we installed self-signed demo certificates)
Enabling OpenSearch Security Plugin
[2022-01-21T20:36:55,927][INFO ][o.o.n.Node               ] [755252261b9b] version[1.2.3], pid[98], build[tar/8a529d77c7432bc45b005ac1c4ba3b2741b57d4a/2021-12-21T01:36:21.407473Z], OS[Linux/5.15.15-200.fc35.x86_64/amd64], JVM[AdoptOpenJDK/OpenJDK 64-Bit Server VM/15.0.1/15.0.1+9]
[2022-01-21T20:36:55,931][INFO ][o.o.n.Node               ] [755252261b9b] JVM home [/usr/share/opensearch/jdk], using bundled JDK [true]
[2022-01-21T20:36:55,931][INFO ][o.o.n.Node               ] [755252261b9b] JVM arguments [-Xshare:auto, -Dopensearch.networkaddress.cache.ttl=60, -Dopensearch.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -XX:+ShowCodeDetailsInExceptionMessages, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dio.netty.allocator.numDirectArenas=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Djava.locale.providers=SPI,COMPAT, -Xms1g, -Xmx1g, -XX:+UseG1GC, -XX:G1ReservePercent=25, -XX:InitiatingHeapOccupancyPercent=30, -Djava.io.tmpdir=/tmp/opensearch-14041432674666057114, -XX:+HeapDumpOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Dclk.tck=100, -Djdk.attach.allowAttachSelf=true, -Djava.security.policy=/usr/share/opensearch/plugins/opensearch-performance-analyzer/pa_config/opensearch_security.policy, -Dopensearch.cgroups.hierarchy.override=/, -XX:MaxDirectMemorySize=536870912, -Dopensearch.path.home=/usr/share/opensearch, -Dopensearch.path.conf=/usr/share/opensearch/config, -Dopensearch.distribution.type=tar, -Dopensearch.bundled_jdk=true]
[2022-01-21T20:36:57,925][INFO ][o.o.s.s.t.SSLConfig      ] [755252261b9b] SSL dual mode is disabled
[2022-01-21T20:36:57,926][INFO ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] OpenSearch Config path is /usr/share/opensearch/config
[2022-01-21T20:36:58,571][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] JVM supports TLSv1.3
[2022-01-21T20:36:58,574][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] Config directory is /usr/share/opensearch/config/, from there the key- and truststore files are resolved relatively
[2022-01-21T20:36:59,663][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] TLS Transport Client Provider : JDK
[2022-01-21T20:36:59,664][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] TLS Transport Server Provider : JDK
[2022-01-21T20:36:59,665][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] TLS HTTP Provider             : JDK
[2022-01-21T20:36:59,665][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] Enabled TLS protocols for transport layer : [TLSv1.3, TLSv1.2, TLSv1.1]
[2022-01-21T20:36:59,666][INFO ][o.o.s.s.DefaultSecurityKeyStore] [755252261b9b] Enabled TLS protocols for HTTP layer      : [TLSv1.3, TLSv1.2, TLSv1.1]
[2022-01-21T20:37:00,218][INFO ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] Clustername: docker-cluster
[2022-01-21T20:37:00,232][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] Directory /usr/share/opensearch/config has insecure file permissions (should be 0700)
[2022-01-21T20:37:00,234][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] File /usr/share/opensearch/config/esnode-key.pem has insecure file permissions (should be 0600)
[2022-01-21T20:37:00,235][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] File /usr/share/opensearch/config/esnode.pem has insecure file permissions (should be 0600)
[2022-01-21T20:37:00,235][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] File /usr/share/opensearch/config/kirk-key.pem has insecure file permissions (should be 0600)
[2022-01-21T20:37:00,236][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] File /usr/share/opensearch/config/root-ca.pem has insecure file permissions (should be 0600)
[2022-01-21T20:37:00,237][WARN ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] File /usr/share/opensearch/config/kirk.pem has insecure file permissions (should be 0600)
[2022-01-21T20:37:00,527][INFO ][o.o.p.c.PluginSettings   ] [755252261b9b] Config: metricsLocation: /dev/shm/performanceanalyzer/, metricsDeletionInterval: 1, httpsEnabled: false, cleanup-metrics-db-files: true, batch-metrics-retention-period-minutes: 7, rpc-port: 9650, webservice-port 9600
[2022-01-21T20:37:01,647][INFO ][o.o.i.r.ReindexPlugin    ] [755252261b9b] ReindexPlugin reloadSPI called
[2022-01-21T20:37:01,649][INFO ][o.o.i.r.ReindexPlugin    ] [755252261b9b] Unable to find any implementation for RemoteReindexExtension
[2022-01-21T20:37:01,681][INFO ][o.o.j.JobSchedulerPlugin ] [755252261b9b] Loaded scheduler extension: opendistro-index-management, index: .opendistro-ism-config
[2022-01-21T20:37:01,722][INFO ][o.o.j.JobSchedulerPlugin ] [755252261b9b] Loaded scheduler extension: reports-scheduler, index: .opendistro-reports-definitions
[2022-01-21T20:37:01,734][INFO ][o.o.j.JobSchedulerPlugin ] [755252261b9b] Loaded scheduler extension: opendistro_anomaly_detector, index: .opendistro-anomaly-detector-jobs
[2022-01-21T20:37:01,745][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [aggs-matrix-stats]
[2022-01-21T20:37:01,746][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [analysis-common]
[2022-01-21T20:37:01,747][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [geo]
[2022-01-21T20:37:01,748][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [ingest-common]
[2022-01-21T20:37:01,749][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [ingest-geoip]
[2022-01-21T20:37:01,749][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [ingest-user-agent]
[2022-01-21T20:37:01,750][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [lang-expression]
[2022-01-21T20:37:01,751][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [lang-mustache]
[2022-01-21T20:37:01,752][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [lang-painless]
[2022-01-21T20:37:01,753][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [mapper-extras]
[2022-01-21T20:37:01,753][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [opensearch-dashboards]
[2022-01-21T20:37:01,754][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [parent-join]
[2022-01-21T20:37:01,754][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [percolator]
[2022-01-21T20:37:01,755][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [rank-eval]
[2022-01-21T20:37:01,755][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [reindex]
[2022-01-21T20:37:01,756][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [repository-url]
[2022-01-21T20:37:01,757][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded module [transport-netty4]
[2022-01-21T20:37:01,759][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-alerting]
[2022-01-21T20:37:01,760][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-anomaly-detection]
[2022-01-21T20:37:01,761][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-asynchronous-search]
[2022-01-21T20:37:01,761][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-cross-cluster-replication]
[2022-01-21T20:37:01,762][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-index-management]
[2022-01-21T20:37:01,762][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-job-scheduler]
[2022-01-21T20:37:01,763][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-knn]
[2022-01-21T20:37:01,763][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-observability]
[2022-01-21T20:37:01,766][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-performance-analyzer]
[2022-01-21T20:37:01,766][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-reports-scheduler]
[2022-01-21T20:37:01,767][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-security]
[2022-01-21T20:37:01,768][INFO ][o.o.p.PluginsService     ] [755252261b9b] loaded plugin [opensearch-sql]
[2022-01-21T20:37:01,811][INFO ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] Disabled https compression by default to mitigate BREACH attacks. You can enable it by setting 'http.compression: true' in opensearch.yml
[2022-01-21T20:37:01,835][DEPRECATION][o.o.d.c.s.Settings       ] [755252261b9b] [node.max_local_storage_nodes] setting was deprecated in OpenSearch and will be removed in a future release! See the breaking changes documentation for the next major version.
[2022-01-21T20:37:01,853][INFO ][o.o.e.NodeEnvironment    ] [755252261b9b] using [1] data paths, mounts [[/ (overlay)]], net usable_space [3.8gb], net total_space [13.3gb], types [overlay]
[2022-01-21T20:37:01,854][INFO ][o.o.e.NodeEnvironment    ] [755252261b9b] heap size [1gb], compressed ordinary object pointers [true]
[2022-01-21T20:37:01,933][INFO ][o.o.n.Node               ] [755252261b9b] node name [755252261b9b], node ID [uJQNvbsQQEa6QU0b13iy3g], cluster name [docker-cluster], roles [master, remote_cluster_client, data, ingest]
[2022-01-21T20:37:05,581][DEPRECATION][o.o.d.c.s.Settings       ] [755252261b9b] [node.max_local_storage_nodes] setting was deprecated in OpenSearch and will be removed in a future release! See the breaking changes documentation for the next major version.
[2022-01-21T20:37:07,724][WARN ][o.o.s.c.Salt             ] [755252261b9b] If you plan to use field masking pls configure compliance salt e1ukloTsQlOgPquJ to be a random string of 16 chars length identical on all nodes
[2022-01-21T20:37:07,774][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Message routing enabled: true
[2022-01-21T20:37:07,880][INFO ][o.o.s.f.SecurityFilter   ] [755252261b9b] <NONE> indices are made immutable.
[2022-01-21T20:37:08,512][INFO ][o.o.a.b.ADCircuitBreakerService] [755252261b9b] Registered memory breaker.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.protostuff.runtime.PolymorphicThrowableSchema (file:/usr/share/opensearch/plugins/opensearch-anomaly-detection/protostuff-runtime-1.7.4.jar) to field java.lang.Throwable.cause
WARNING: Please consider reporting this to the maintainers of io.protostuff.runtime.PolymorphicThrowableSchema
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[2022-01-21T20:37:09,494][INFO ][o.o.t.NettyAllocator     ] [755252261b9b] creating NettyAllocator with the following configs: [name=unpooled, suggested_max_allocation_size=256kb, factors={opensearch.unsafe.use_unpooled_allocator=null, g1gc_enabled=true, g1gc_region_size=1mb, heap_size=1gb}]
[2022-01-21T20:37:09,657][INFO ][o.o.d.DiscoveryModule    ] [755252261b9b] using discovery type [single-node] and seed hosts providers [settings]
[2022-01-21T20:37:10,653][WARN ][o.o.g.DanglingIndicesState] [755252261b9b] gateway.auto_import_dangling_indices is disabled, dangling indices will not be automatically detected or imported and must be managed manually
[2022-01-21T20:37:11,211][INFO ][o.o.p.h.c.PerformanceAnalyzerConfigAction] [755252261b9b] PerformanceAnalyzer Enabled: false
[2022-01-21T20:37:11,430][INFO ][o.o.n.Node               ] [755252261b9b] initialized
[2022-01-21T20:37:11,431][INFO ][o.o.n.Node               ] [755252261b9b] starting ...
[2022-01-21T20:37:11,697][INFO ][o.o.t.TransportService   ] [755252261b9b] publish_address {10.0.2.100:9300}, bound_addresses {[::]:9300}
[2022-01-21T20:37:11,891][WARN ][o.o.b.BootstrapChecks    ] [755252261b9b] max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
[2022-01-21T20:37:11,911][INFO ][o.o.c.c.Coordinator      ] [755252261b9b] setting initial configuration to VotingConfiguration{uJQNvbsQQEa6QU0b13iy3g}
[2022-01-21T20:37:12,135][INFO ][o.o.c.s.MasterService    ] [755252261b9b] elected-as-master ([1] nodes joined)[{755252261b9b}{uJQNvbsQQEa6QU0b13iy3g}{TmZjDDaMRFK32BTvfY4XJg}{10.0.2.100}{10.0.2.100:9300}{dimr}{shard_indexing_pressure_enabled=true} elect leader, _BECOME_MASTER_TASK_, _FINISH_ELECTION_], term: 1, version: 1, delta: master node changed {previous [], current [{755252261b9b}{uJQNvbsQQEa6QU0b13iy3g}{TmZjDDaMRFK32BTvfY4XJg}{10.0.2.100}{10.0.2.100:9300}{dimr}{shard_indexing_pressure_enabled=true}]}
[2022-01-21T20:37:12,210][INFO ][o.o.c.c.CoordinationState] [755252261b9b] cluster UUID set to [oJr_PrzZS72xXdeYH6zdEw]
[2022-01-21T20:37:12,226][INFO ][o.o.c.s.ClusterApplierService] [755252261b9b] master node changed {previous [], current [{755252261b9b}{uJQNvbsQQEa6QU0b13iy3g}{TmZjDDaMRFK32BTvfY4XJg}{10.0.2.100}{10.0.2.100:9300}{dimr}{shard_indexing_pressure_enabled=true}]}, term: 1, version: 1, reason: Publication{term=1, version=1}
[2022-01-21T20:37:12,262][WARN ][o.o.p.c.s.h.ConfigOverridesClusterSettingHandler] [755252261b9b] Config override setting update called with empty string. Ignoring.
[2022-01-21T20:37:12,265][INFO ][o.o.a.c.ADClusterEventListener] [755252261b9b] Cluster is not recovered yet.
[2022-01-21T20:37:12,311][INFO ][o.o.h.AbstractHttpServerTransport] [755252261b9b] publish_address {10.0.2.100:9200}, bound_addresses {[::]:9200}
[2022-01-21T20:37:12,312][INFO ][o.o.n.Node               ] [755252261b9b] started
[2022-01-21T20:37:12,313][INFO ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] Node started
[2022-01-21T20:37:12,314][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Will attempt to create index .opendistro_security and default configs if they are absent
[2022-01-21T20:37:12,316][INFO ][o.o.s.OpenSearchSecurityPlugin] [755252261b9b] 0 OpenSearch Security modules loaded so far: []
[2022-01-21T20:37:12,317][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Background init thread started. Install default config?: true
[2022-01-21T20:37:12,373][INFO ][o.o.a.c.HashRing         ] [755252261b9b] Node added: [uJQNvbsQQEa6QU0b13iy3g]
[2022-01-21T20:37:12,397][INFO ][o.o.a.c.HashRing         ] [755252261b9b] Add data node to AD version hash ring: uJQNvbsQQEa6QU0b13iy3g
[2022-01-21T20:37:12,417][INFO ][o.o.a.c.HashRing         ] [755252261b9b] All nodes with known AD version: {uJQNvbsQQEa6QU0b13iy3g=ADNodeInfo{version=1.2.3, isEligibleDataNode=true}}
[2022-01-21T20:37:12,418][INFO ][o.o.a.c.HashRing         ] [755252261b9b] Rebuild AD hash ring for realtime AD with cooldown, nodeChangeEvents size 0
[2022-01-21T20:37:12,419][INFO ][o.o.a.c.HashRing         ] [755252261b9b] Build AD version hash ring successfully
[2022-01-21T20:37:12,421][INFO ][o.o.g.GatewayService     ] [755252261b9b] recovered [0] indices into cluster_state
[2022-01-21T20:37:12,430][INFO ][o.o.a.c.ADDataMigrator   ] [755252261b9b] Start migrating AD data
[2022-01-21T20:37:12,431][INFO ][o.o.a.c.ADDataMigrator   ] [755252261b9b] AD job index doesn't exist, no need to migrate
[2022-01-21T20:37:12,432][INFO ][o.o.a.c.ADClusterEventListener] [755252261b9b] Init AD version hash ring successfully
[2022-01-21T20:37:12,691][INFO ][o.o.c.m.MetadataCreateIndexService] [755252261b9b] [.opendistro_security] creating index, cause [api], templates [], shards [1]/[1]
[2022-01-21T20:37:12,705][INFO ][o.o.c.r.a.AllocationService] [755252261b9b] updating number_of_replicas to [0] for indices [.opendistro_security]
[2022-01-21T20:37:12,990][INFO ][o.o.c.r.a.AllocationService] [755252261b9b] Cluster health status changed from [YELLOW] to [GREEN] (reason: [shards started [[.opendistro_security][0]]]).
[2022-01-21T20:37:13,041][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Index .opendistro_security created?: true
[2022-01-21T20:37:13,042][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Node started, try to initialize it. Wait for at least yellow cluster state....
[2022-01-21T20:37:13,046][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'config' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/config.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,223][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] create_mapping [_doc]
[2022-01-21T20:37:13,386][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'config' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,386][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'roles' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/roles.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,421][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,482][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'roles' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,483][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'rolesmapping' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/roles_mapping.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,524][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,598][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'rolesmapping' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,599][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'internalusers' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/internal_users.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,624][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,689][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'internalusers' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,690][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'actiongroups' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/action_groups.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,722][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,784][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'actiongroups' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,786][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'tenants' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/tenants.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:13,802][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,850][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'tenants' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,851][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'nodesdn' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/nodes_dn.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=true
[2022-01-21T20:37:13,882][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:13,951][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'nodesdn' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:13,952][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'whitelist' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/whitelist.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=true
[2022-01-21T20:37:13,966][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:14,019][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'whitelist' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:14,020][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Will update 'audit' with /usr/share/opensearch/plugins/opensearch-security/securityconfig/audit.yml and populate it with empty doc if file missing and populateEmptyIfFileMissing=false
[2022-01-21T20:37:14,062][INFO ][o.o.c.m.MetadataMappingService] [755252261b9b] [.opendistro_security/hBXfg7FcTA6FmoO-qK9tIA] update_mapping [_doc]
[2022-01-21T20:37:14,129][INFO ][o.o.s.s.ConfigHelper     ] [755252261b9b] Doc with id 'audit' and version 2 is updated in .opendistro_security index.
[2022-01-21T20:37:14,349][INFO ][stdout                   ] [755252261b9b] [FINE] No subscribers registered for event class org.opensearch.security.securityconf.DynamicConfigFactory$NodesDnModelImpl
[2022-01-21T20:37:14,350][INFO ][stdout                   ] [755252261b9b] [FINE] No subscribers registered for event class org.greenrobot.eventbus.NoSubscriberEvent
[2022-01-21T20:37:14,352][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing on REST API is enabled.
[2022-01-21T20:37:14,355][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] [AUTHENTICATED, GRANTED_PRIVILEGES] are excluded from REST API auditing.
[2022-01-21T20:37:14,356][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing on Transport API is enabled.
[2022-01-21T20:37:14,356][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] [AUTHENTICATED, GRANTED_PRIVILEGES] are excluded from Transport API auditing.
[2022-01-21T20:37:14,357][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing of request body is enabled.
[2022-01-21T20:37:14,358][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Bulk requests resolution is disabled during request auditing.
[2022-01-21T20:37:14,358][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Index resolution is enabled during request auditing.
[2022-01-21T20:37:14,359][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Sensitive headers auditing is enabled.
[2022-01-21T20:37:14,359][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing requests from kibanaserver users is disabled.
[2022-01-21T20:37:14,371][WARN ][o.o.s.a.r.AuditMessageRouter] [755252261b9b] No endpoint configured for categories [BAD_HEADERS, FAILED_LOGIN, MISSING_PRIVILEGES, GRANTED_PRIVILEGES, OPENDISTRO_SECURITY_INDEX_ATTEMPT, SSL_EXCEPTION, AUTHENTICATED, INDEX_EVENT, COMPLIANCE_DOC_READ, COMPLIANCE_DOC_WRITE, COMPLIANCE_EXTERNAL_CONFIG, COMPLIANCE_INTERNAL_CONFIG_READ, COMPLIANCE_INTERNAL_CONFIG_WRITE], using default endpoint
[2022-01-21T20:37:14,372][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing of external configuration is disabled.
[2022-01-21T20:37:14,372][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing of internal configuration is enabled.
[2022-01-21T20:37:14,373][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing only metadata information for read request is enabled.
[2022-01-21T20:37:14,373][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing will watch {} for read requests.
[2022-01-21T20:37:14,374][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing read operation requests from kibanaserver users is disabled.
[2022-01-21T20:37:14,375][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing only metadata information for write request is enabled.
[2022-01-21T20:37:14,376][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing diffs for write requests is disabled.
[2022-01-21T20:37:14,376][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing write operation requests from kibanaserver users is disabled.
[2022-01-21T20:37:14,377][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Auditing will watch <NONE> for write requests.
[2022-01-21T20:37:14,379][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] .opendistro_security is used as internal security index.
[2022-01-21T20:37:14,380][INFO ][o.o.s.a.i.AuditLogImpl   ] [755252261b9b] Internal index used for posting audit logs is null
[2022-01-21T20:37:14,381][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Hot-reloading of audit configuration is enabled
[2022-01-21T20:37:14,382][INFO ][o.o.s.c.ConfigurationRepository] [755252261b9b] Node '755252261b9b' initialized
```

The output of _podman image ls_ now is: 

```
REPOSITORY                              TAG         IMAGE ID      CREATED      SIZE
docker.io/opensearchproject/opensearch  latest      9ae13ad440d3  3 days ago   834 MB
docker.io/opensearchproject/opensearch  1.2.3       e72207c604f2  4 weeks ago  845 MB
```

Thanks a lot again!!

--- a/docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh
+++ b/docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh
@@ -160,8 +160,12 @@ function setupSecurityDashboardsPlugin {
         if [ "$DISABLE_SECURITY_DASHBOARDS_PLUGIN" = "true" ]; then
             echo "Disabling OpenSearch Security Dashboards Plugin"
             ./bin/opensearch-dashboards-plugin remove securityDashboards
-            cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
-            cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "s/https/http/g" | tee $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
+
+            # Remove all security related parameters as well as changing HTTPS to HTTP
+            # Temporary fix before security-dashboards plugin implement a parameter to disable the plugin entirely
+            # https://github.com/opensearch-project/security-dashboards-plugin/issues/896
+            NEW_CONFIG=`cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | sed "s/https/http/g"`
+            echo "$NEW_CONFIG" > $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
         fi
     fi
 }

--- a/docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh
+++ b/docker/release/config/opensearch-dashboards/opensearch-dashboards-docker-entrypoint.sh
@@ -164,8 +164,8 @@ function setupSecurityDashboardsPlugin {
             # Remove all security related parameters as well as changing HTTPS to HTTP
             # Temporary fix before security-dashboards plugin implement a parameter to disable the plugin entirely
             # https://github.com/opensearch-project/security-dashboards-plugin/issues/896
-            NEW_CONFIG=`cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | sed "s/https/http/g"`
-            echo "$NEW_CONFIG" > $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
+            UPDATED_CONFIG=`cat $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml | sed "/^opensearch_security/d" | sed "s/https/http/g"`
+            echo "$UPDATED_CONFIG" > $OPENSEARCH_DASHBOARDS_HOME/config/opensearch_dashboards.yml
         fi
     fi
 }

--- a/docker/release/config/opensearch/opensearch-docker-entrypoint.sh
+++ b/docker/release/config/opensearch/opensearch-docker-entrypoint.sh
@@ -43,11 +43,10 @@ function setupSecurityPlugin {
 
         if [ "$DISABLE_SECURITY_PLUGIN" = "true" ]; then
             echo "Disabling OpenSearch Security Plugin"
-            cat $OPENSEARCH_HOME/config/opensearch.yml | sed "/plugins.security.disabled/d" | tee $OPENSEARCH_HOME/config/opensearch.yml
-            echo "plugins.security.disabled: true" >> $OPENSEARCH_HOME/config/opensearch.yml
+            opensearch_opt="-Eplugins.security.disabled=true"
+            opensearch_opts+=("${opensearch_opt}")
         else
             echo "Enabling OpenSearch Security Plugin"
-            cat $OPENSEARCH_HOME/config/opensearch.yml | sed "/plugins.security.disabled/d" | tee $OPENSEARCH_HOME/config/opensearch.yml
         fi
     fi
 }
@@ -82,7 +81,7 @@ function runOpensearch {
     #
     # e.g. Setting the env var cluster.name=testcluster
     # will cause OpenSearch to be invoked with -Ecluster.name=testcluster
-    local opensearch_opts=()
+    opensearch_opts=()
     while IFS='=' read -r envvar_key envvar_value
     do
         # OpenSearch settings need to have at least two dot separated lowercase

